easybuild/easyconfigs/j/Jansson/Jansson-2.14.1-GCCcore-13.2.0.eb

```
@@ -0,0 +1,40 @@
+# Contribution from Imperial College London
+# uploaded by J. Sassmannshausen
+# Update: P.Tománek (Inuits)
+# Update: A. O'Cais
+
+easyblock = 'CMakeMake'
+
+name = 'Jansson'
+version = '2.14.1'
+
+homepage = 'https://www.digip.org/jansson/'
+description = """Jansson is a C library for encoding, decoding and manipulating JSON data.
+ Its main features and design principles are:
+ * Simple and intuitive API and data model
+ * Comprehensive documentation
+ * No dependencies on other libraries
+ * Full Unicode support (UTF-8)
+ * Extensive test suite"""
+
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+
+source_urls = ['https://github.com/akheron/%(namelower)s/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['979210eaffdffbcf54cfc34d047fccde13f21b529a381df26db871d886f729a4']
+
+builddependencies = [
+    ('CMake', '3.27.6'),
+    ('binutils', '2.40'),
+]
+
+configopts = "-DJANSSON_BUILD_SHARED_LIBS=ON "
+
+runtest = 'check'
+
+sanity_check_paths = {
+    'files': ['lib/libjansson.so'],
```

From the diff with `Jansson-2.14-GCC-12.3.0.eb`

```suggestion
    'files': ['lib/libjansson.%s' % SHLIB_EXT],
```